deepchecks/tabular/checks/model_evaluation/performance_report.py

```diff
@@ -150,6 +156,7 @@ def reduce_output(self, check_result: CheckResult) -> Dict[str, float]:
         """Return the values of the metrics for the test dataset in {metric: value} format."""
         df = check_result.value
         df = df[df['Dataset'] == DatasetKind.TEST.value]
+        df = df.groupby('Metric').aggregate(self.reduce).reset_index()
```

Comment:
the aggregate function accepts strings and functions?

Reply:
yas, even tested